modules/samtools/collatefastq/main.nf

```diff
@@ -0,0 +1,47 @@
+process SAMTOOLS_COLLATEFASTQ {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::samtools=1.15.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/samtools:1.15.1--h1170115_0' :
+        'quay.io/biocontainers/samtools:1.15.1--h1170115_0' }"
+
+    input:
+    tuple val(meta), path(input)
+
+    output:
+    //TODO might be good to have ordered output of the fastq files, so we can
```

Comment:
How about having separate output channesl for paired, singletons, and other?

Reply:
I was just thinking having separate output channels is easier then sorting it, or otherwise explicitly? :
```
meta, 1.read, 2.read, singleton, other , emit: reads
```
?

Reply:
I'm on it